Source/ACE/Entity/Landblock.cs

```diff
@@ -396,26 +396,44 @@ private void UpdateStatus(int pcount)
             }
         }
 
+        /// <summary>
+        /// Gets all landblocks in range of a position.  (for indoors positions that is just this landblock)
+        /// </summary>
+        /// <param name="pos"></param>
+        /// <param name="distance"></param>
+        /// <returns></returns>
         private List<Landblock> GetLandblocksInRange(Position pos, float distance)
         {
             List<Landblock> inRange = new List<Landblock>();
 
             inRange.Add(this);
 
+            if (pos.Indoors)
+            {
```

Comment:
FWIW, this is worth testing when you walk in a building.  I think that flag ends up set in a building in the Overworld.